### DISCUSSION TITLE
Forking your own repository

### DISCUSSION THREAD

**Discussion by JJTech0130** [community/community#20903] [Other Feature Feedback, Questions, & Ideas]
I noticed that organizational accounts can fork their own repositories-- so why can't personal accounts?

For example, in a repository *owned by an organization*, I can fork it *to that organization*: simply changing to something like `originalname-1` so the names don't conflict:
![Org account](https://user-images.githubusercontent.com/53275876/179857174-98240dd9-ffc4-4492-b5ca-f92c4ddf74aa.png)
But in a repository *owned by a personal account*, the option to fork it *to that personal account* is greyed out with "repository already exists":
![Personal account](https://user-images.githubusercontent.com/53275876/179857116-a2c95a0c-e90d-4a3b-80fc-53eb75d30c41.png)

This appears to be an intended feature as Microsoft uses it: https://github.com/microsoft/fluentui-apple-1

**Comment by jfullstackdev** [comment]
my own opinion is that, Orgs are composed of members while a personal account is not, there is no sense if you will be forking your own personal repo, you can simply create a branch, but in a large scale collaboration, that will matter, hence the ability to be forked within that org

it's just that, we are allowed to create Orgs even without members but the functionality is still there by default
